### DISCUSSION TITLE
Does jaeger support cgroup v2?

### DISCUSSION THREAD

**Discussion by sidharthabiswal** [jaegertracing/jaeger#4938] [Q&A]
Does jaeger support cgroup v2? Is there any reference link to get more information about this?

**Comment by yurishkuro** [comment]
What does supporting cgroup mean to you?

**Comment by sidharthabiswal** [comment]
Hi @yurishkuro,

I mean will the Jaeger code be able to run with cgroups v2?

**Comment by sidharthabiswal** [comment]
Hi @yurishkuro,

When we grep for cgroups, we get some results in "jaeger-collector" and "jaeger-query" binaries.
But we are not sure if both v1 and v2 code have been implemented as mentioned in https://pkg.go.dev/github.com/containerd/cgroups.
Could you please share your opinion?
